src/utilities/Validation.ts

```diff
@@ -0,0 +1,17 @@
+export const isFieldRequired = {
+  value: true,
+  message: 'This field is required',
+};
+
+export const isEmailValid = {
+  value: /\S+@\S+\.\S+/,
+  message: 'E-mail is not valid',
+};
+
+export const isMinValue = (value:string | undefined, min:number) => {
```

Comment:
```suggestion
export const isMinValue = (value?: string, min:number) => {
```

Reply:
cannot be set this way

unless so (value: string = '', min:number)